routes/index.js

```diff
@@ -96,6 +94,19 @@ exports.conversation = function (req, res) {
 	});
 }
 
+exports.account = function (req, res){
+	if (!req.user || !req.user._id) {
+		res.render('error', {
+			title: "Looks like you're not logged in!"
+		});
+		return;
+	} else{
+		res.render('account', {
+			user: req.user
+		});
+	}
+}
```

Comment:
Just want to clarify what this code does for my own understanding. Is this used to handle errors when the user clicks the modal for their account info?